vsts/vsts.yaml

```diff
@@ -233,7 +233,7 @@ jobs:
 
     condition: succeeded()
     pool:
-      name: Hosted VS2017
+      name: vs2017-win2016
```

Comment:
Changed this to the correct Azure Pipelines name. After talking to someone from the PoliCheck engineering group it was suggested we use the same hosted pipeline as they do.